monai/transforms/intensity/dictionary.py

```diff
@@ -110,27 +110,29 @@ def __init__(
     ) -> None:
         super().__init__(keys)
         self.prob = prob
-        self.mean = ensure_tuple_size(mean, len(self.keys))
+        self.mean = ensure_tuple_size(mean, len(self.keys), mean)
```

Comment:
I think for this case we should use `ensure_tuple_rep(mean, len(self.keys))`, right?
For example: https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/utility/dictionary.py#L294

Thanks.

Reply:
Yup, thanks!